### PR TITLE
Defined new REST API to get artifacts in a given environment

### PR DESCRIPTION
**Pull request by maheshkelkar**
References #21

### CODE REVIEW

**Comment by maheshkelkar** [comment]
@GiriDandu Please take a look at latest rev

File: features/api/environment/reading.feature [hunk 1]
```diff
@@ -72,3 +72,91 @@ Feature: Environment READ APIs
...
+          "infoUrl" : null,
+          "createdAt" : "{{created_timestamp}}"
+         }],
+        "createdAt" : "{{created_timestamp}}"
+      }]
+    """
+
+
+  @freezetime @wip
+  Scenario: Fetching all deployments in the environment by the pageNumber and perPageSize
```
Comment (GiriDandu) [inline features/api/environment/reading.feature]: Can you add test for environment that doesn't exist. Should return 404 in that case

**Comment by rtyler** [comment]
:+1:

File: features/api/environment/reading.feature (hunk 1 above)
Comment (maheshkelkar) [inline features/api/environment/reading.feature]: done

**Comment by maheshkelkar** [comment]
Thanks. I will merge when Travis is green.

**Pull request merged**